aea/cli/utils/decorators.py

```diff
@@ -114,20 +114,34 @@ def _validate_config_consistency(ctx: Context, check_aea_version: bool = True) -
         )
 
 
-def _check_aea_project(args: Tuple[Any, ...], check_aea_version: bool = True) -> None:
+def _check_aea_project(
+    args: Tuple[Any, ...],
+    check_aea_version: bool = True,
+    check_finger_prints: bool = False,
+) -> None:
     try:
         click_context = args[0]
         ctx = cast(Context, click_context.obj)
         try_to_load_agent_config(ctx)
         skip_consistency_check = ctx.config["skip_consistency_check"]
         if not skip_consistency_check:
             _validate_config_consistency(ctx, check_aea_version=check_aea_version)
+        if check_finger_prints:
```

Comment:
It's false everywhere? Why was it added?

Reply:
it's used for tests at least, and probably will be used in future as a part of consistency check